Gems/ROS2/Code/Source/VehicleDynamics/ModelLimits/AckermannModelLimits.cpp

```diff
@@ -34,7 +35,11 @@ namespace ROS2::VehicleDynamics
                     ->DataElement(
                         AZ::Edit::UIHandlers::Default, &AckermannModelLimits::m_steeringLimit, "Steering Limit", "Max steering angle (rad)")
                     ->Attribute(AZ::Edit::Attributes::Min, 0.0f)
-                    ->Attribute(AZ::Edit::Attributes::Max, 1.57f);
+                    ->Attribute(AZ::Edit::Attributes::Max, 1.57f)
+                    ->DataElement(
+                        AZ::Edit::UIHandlers::Default, &AckermannModelLimits::m_accelearation, "Acceleration", "Acceleration in m/s²")
```

Comment:
Can you confirm that the '²' symbol renders in the component properly?

Reply:
![Screenshot from 2023-02-02 15-46-06](https://user-images.githubusercontent.com/3209244/216662570-9b133f09-7bbd-471c-aae8-ab2c390c268b.png)
It works fine in my environment.